charts/osdfir-infrastructure/charts/grr/templates/job-build-grr-client.yaml

```diff
@@ -0,0 +1,97 @@
+{{- $frontend := "" -}}
+{{- if or (eq .Values.fleetspeak.frontend.expose "internal") (eq .Values.fleetspeak.frontend.expose "external") -}}
+  {{- $frontend = .Values.fleetspeak.frontend.address -}}
+{{- else }}
+  {{- /* Step 1: Perform lookup, store result */}}
+  {{- $lookupResult := lookup "v1" "Node" "" "" | default dict -}}
+  {{- /* Step 2: Get values, store result */}}
+  {{- $nodeList := $lookupResult.items -}}
+  {{- if $nodeList -}}
+    {{- /* Step 3: Get first node data, store result */}}
+    {{- $nodeData := first $nodeList -}}
+    {{- /* Step 4: Get the status */}}
+    {{- $statusValue := $nodeData.status -}}
+    {{- /* Step 5: Get the addresses */}}
+    {{- $addressesList := $statusValue.addresses -}}
+    {{- /* Step 6: Get the InternalIP address */}}
+    {{- range $address := $addressesList -}}
+      {{- if and (not $frontend) (eq $address.type "InternalIP") -}}
+        {{- $frontend = $address.address -}}
+      {{- end -}}
+    {{- end -}}
+  {{- end -}}
+{{- end -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-grr-client-builder
```

Comment:
s/job-grr-client-builder/{{.Release.Name}}-grr-job-client-builder

Reply:
Also for mysql if we could do the same please

s/mysql/{{.Release-Name}}-grr-mysql

Reply:
Updated accordingly